eip.tf

```diff
@@ -1,12 +1,8 @@
 #Elastic IP nat
 resource aws_eip "nat" {
-count = length(var.subnets_cidr_private)
-vpc = true
-}
+  count = length(var.subnets_cidr_private)
```

Comment:
You did not move the values inward. (tab)
The resource block and its values cannot be at the same level. it will not work.